Orange/widgets/visualize/tests/test_ownomogram.py

```diff
@@ -23,8 +23,7 @@ def setUpClass(cls):
         cls.lenses = Table("lenses")
 
     def setUp(self):
-        self.widget = self.create_widget(OWNomogram)
-        self.widget.repaint = True
+        self.widget = self.create_widget(OWNomogram)  # type: OWNomogram
 
```

Comment:
The two length-asserting test failures can be mitigated by setting in `setUp()`:
```py
self.widget.display_index = 0  # i.e. display all attributes
```
Anybody has an idea of an alternative solution, and perhaps something for the third failing test as well?